backend/internal/server/apis/rsvp/ref/rsvp.go

```diff
@@ -46,6 +52,9 @@ func handleRef(c *fiber.Ctx) error {
 		refSchedule = append(refSchedule, row)
 	}
 
-	csv.GenerateCsv(refSchedule, "ref_schedule.csv")
+	err := csv.GenerateCsv(refSchedule, "ref_schedule.csv")
```

Comment:
Is the output of our generate function going to stay csv? I dont' think it needs to be CSV anymore, we should be able to simply update the DB

Reply:
Yeah we dont but for now we still have it. I will work on getting rid of it after Jacob talks about the DB in class tomorrow.